src/js/puzzles.js

```diff
@@ -333,28 +333,28 @@ var puzzlesArray = [
   id: 48,
   sentence: 'Are you sure this will be sun?',
   word: 'sun',
-  hint: 'Time of my life',
+  hint: 'time of my life',
   answer: 'fun'
 },
 {
   id: 49,
   sentence: 'The star is out of my peach.',
   word: 'peach',
-  hint: 'Within arms-length',
+  hint: 'within arms-length',
   answer: 'reach'
 },
 {
   id: 50,
   sentence: 'Ferry pie makes for an amazing treat.',
   word: 'ferry',
-  hint: 'small, red fruit',
+  hint: 'a small red fruit',
```

Comment:
In English, formal grammar dictates adjectives are separated by a comma, so `small, red fruit` is correct.

Reply:
Fixed on my latest commit.